pkg/trait/jvm.go

```diff
@@ -54,6 +54,8 @@ type jvmTrait struct {
 	DebugAddress string `property:"debug-address" json:"debugAddress,omitempty"`
 	// A list of JVM options
 	Options []string `property:"options" json:"options,omitempty"`
+	// A list of additional JVM classpaths
+	Classpaths []string `property:"classpaths" json:"classpaths,omitempty"`
```

Comment:
What about reflecting the usual classpath setting, that is adding a string property, and follow https://docs.oracle.com/javase/8/docs/technotes/tools/windows/classpath.html?

Reply:
Yeah, the only concern that I have there is that `cp` syntax is different for each operating system. A `Windows` user may be using a `;` resulting in a runtime error because not recognized by `Linux`. Maybe we can just document it and leave it as a single property (I didn't like the plural `classpaths` either...).

Reply:
Right, I think it's a power users use case, so we can assume they know what they are doing, and understand Linux style separator must be used.